.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -592,6 +592,37 @@ private DateTimeResolutionResult ParseImplicitDate(string text, DateObject refer
                 return ret;
             }
 
+            if ((config.Options & DateTimeOptions.TasksMode) != 0)
+            {
+                trimmedText = text.Trim();
+                if (trimmedText.Contains("next week"))
```

Comment:
Don't hard code natural language words. These should be defined as resources in YAML and there are probably resources that can be re-used (i.e., don't declare repeated ones). Also, can't this be a method? Ideally, if it doesn't depend on the parser state, even moded to the task-specific utils.

Reply:
A short code comment, here on where the method is declared, explaining the resolution difference would also be helpful.

Reply:
Updated YAML with required regex.
I have extracted datetime reference as part of implicit date regex, so it's parsing is part of the implicitdate parsing method, since it's language dependent so haven't shifted it into TaskModeProcessing.cs file.